Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/CriticalLogsAnalyticsHandler.cs

```diff
@@ -30,38 +34,40 @@ private void TrackUnhandledException(object sender, UnhandledExceptionEventArgs
                 { "type", "unhandled exception" },
                 { "category", IAnalyticsController.UNDEFINED },
                 { "scene_hash", IAnalyticsController.UNDEFINED },
-                { "message", e.Message },
+                { "message", IsPayloadSizeValid(e.Message) ? e.Message : LONG_MESSAGE_PLACEHOLDER },
             });
         }
 
-        public void Log(LogType logType, ReportData reportData, Object context, object message)
+        public void Log(LogType logType, ReportData reportData, Object context, object messageObj)
         {
-            if(logType != LogType.Error && logType != LogType.Exception) return;
+            if (logType != LogType.Error && logType != LogType.Exception) return;
+
+            var message = messageObj.ToString();
 
             analytics.Track(AnalyticsEvents.General.ERROR, new JsonObject
             {
                 { "type", logType.ToString() },
                 { "category", reportData.Category },
                 { "scene_hash", reportData.SceneShortInfo.Name },
-                { "message", message.ToString() },
+                { "message", IsPayloadSizeValid(message) ? message : LONG_MESSAGE_PLACEHOLDER },
             });
         }
 
         public void LogFormat(LogType logType, ReportData reportData, Object context, object message, params object[] args)
         {
-            if(logType != LogType.Error && logType != LogType.Exception) return;
+            if (logType != LogType.Error && logType != LogType.Exception) return;
 
             Log(logType, reportData, context, string.Format(message.ToString(), args));
         }
 
-        public void LogException<T>(T ecsSystemException) where T : Exception, IDecentralandException
+        public void LogException<T>(T ecsSystemException) where T: Exception, IDecentralandException
         {
             analytics.Track(AnalyticsEvents.General.ERROR, new JsonObject
             {
                 { "type", "exception" },
                 { "category", "ecs" },
                 { "scene_hash", IAnalyticsController.UNDEFINED },
-                { "message", ecsSystemException.Message },
+                { "message", IsPayloadSizeValid(ecsSystemException.Message) ? ecsSystemException.Message : LONG_MESSAGE_PLACEHOLDER },
```

Comment:
Wouldnt you rather send a truncated message rather than the `LONG_MESSAGE_PLACEHOLDER`?